src/parsec_service/test.md

```diff
@@ -100,4 +100,75 @@ target and then observing the code segments reached using said input. To stop th
 crashes or slow tests, as well as tests that lead to a timeout, are stored in
 `./fuzz/artifacts/fuzz_service/`.
 
+## Testing the TPM provider using the Software TPM
+
+If you do not have a hardware TPM, or you do not want to use it for tests, you might want to use the
+IBM Software TPM. You will need to follow these steps:
+
+- install the TSS libraries ([installation
+   guide](https://github.com/tpm2-software/tpm2-tss/blob/master/INSTALL.md)). Parsec has been tested
+   with version 2.3.3.
+- install the [TPM tools](https://github.com/tpm2-software/tpm2-tools)
+- install the [Software
+   TPM](https://sourceforge.net/projects/ibmswtpm2/files/ibmtpm1332.tar.gz/download), run `make` in
```

Comment:
Is it possible to point that URL to a more generic link? I'm afraid of forgetting to ever update it since they do sometimes push new versions out.

Reply:
Sure! I will change it to something more generic. On the same topic, there is now a new software TPM used by default by the stack that we use: https://github.com/tpm2-software/tpm2-tss#software-tpm